pkg/workflow/dataimpl.go

```diff
@@ -134,3 +251,56 @@ func (d *DataImpl) GetErrorList() []snyk_errors.Error {
 func (d *DataImpl) AddError(err snyk_errors.Error) {
 	d.errors = append(d.errors, err)
 }
+
+func (d *DataImpl) SetConfiguration(config configuration.Configuration) {
+	d.configuration = config
+}
+
+func (d *DataImpl) GetConfiguration() configuration.Configuration {
+	return d.configuration
+}
+
+func setPayloadLocation(id Identifier, config configuration.Configuration, payload interface{}, logger *zerolog.Logger) Location {
+	payloadLocation := Location{
+		Path: "",
+		Type: InMemory,
+	}
+
+	logger.Debug().Msg("checking if payload is []byte")
+	bytes, ok := payload.([]byte)
+	if !ok {
+		return payloadLocation
+	}
+
+	payloadSize := len(bytes)
+	fileSizeThreshold := config.GetInt(configuration.IN_MEMORY_THRESHOLD_BYTES)
+	logger.Debug().Msgf("payload is []byte, comparing payload size (%d bytes) to threshold (%d bytes)", payloadSize, fileSizeThreshold)
+
+	if payloadSize <= fileSizeThreshold {
```

Comment:
Question: how can I disable it through configuration. Setting a large number? how about setting the threshold to -1.